bindings/src/main/java/io/pravega/storage/extendeds3/ExtendedS3Storage.java

```diff
@@ -310,13 +322,20 @@ private SegmentHandle doCreate(String streamSegmentName) throws StreamSegmentExi
         }
         client.putObject(request);
 
+        Duration elapsed = timer.getElapsed();
+
+        ExtendedS3Metrics.CREATE_LATENCY.reportSuccessEvent(elapsed);
+        ExtendedS3Metrics.CREATE_COUNT.inc();
+
+        log.debug("Create  segment={} latency={}.", elapsed);
```

Comment:
here too

Plus you are missing an arg for the segment.

Reply:
Good catch. Thanks.
Fixed.